File: despatch.WEB/app/src/swagger/models/contact-data-addresses-request.ts
/* tslint:disable */
import { BaseEntityRequest } from './base-entity-request';
import { AddressRequest } from './address-request';
import { ContactDataRequest } from './contact-data-request';
export interface ContactDataAddressesRequest extends BaseEntityRequest{
  address?: AddressRequest;
  addressId?: string;
  contactData?: ContactDataRequest;
  contactDataId?: string;
}

File: despatch.Domain/Models/Default/Request/ContactDataAddressesRequest.cs
namespace despatch.Domain.Models.Default.Request
{
    public class ContactDataAddressesRequest : BaseEntityRequest
    {
        public string AddressId { get; set; }
        public string ContactDataId { get; set; }

        public virtual AddressRequest Address { get; set; }
        public virtual ContactDataRequest ContactData { get; set; }
    }
}
